internal/runner/service_test.go

```diff
@@ -568,19 +568,45 @@ func Test_runnerService(t *testing.T) {
 	t.Run("ExecuteLicenseCat", func(t *testing.T) {
 		t.Parallel()
 
+		session, _ := client.CreateSession(context.Background(), &runnerv1.CreateSessionRequest{})
+		sessionID := session.Session.Id
+
 		stream, err := client.Execute(context.Background())
 		require.NoError(t, err)
 
+		err = stream.Send(&runnerv1.ExecuteRequest{
+			ProgramName: "bash",
+			Directory:   "../..",
+			Commands: []string{
+				"export LICENSE=$(cat LICENSE)",
+			},
+			SessionId: sessionID,
+		})
+		assert.NoError(t, err)
```

Comment:
I use `require` when there is no point in moving further if a condition is false. `assert` is used when it's ok to continue.